### DISCUSSION TITLE
OSS主页机子被PCDN封了就离谱撒币联通.jpg

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#5154] [讨论]
~~机子换了地方放.jpg 凌云还不知道这件事 解析还没更新...~~

~~机房其他机子因为跑MC Mod服导致高上传被锁机房了~~

临时访问done 等待机房审核

预计影响
杂志主页
新闻速报[预设地址正常访问 新地址暂停访问]

**Comment by wuliaodexiaoluo** [comment]
你干脆让凌云 CNAME 到面板地址或者让服务商单独提供一个地址 CNAME 得了（（（

**Reply by wuliaodexiaoluo** [reply]
不然每次换地方都得重新上解析，还要等十分钟同步解析

**Comment by LinQingYuu** [comment]
>PCDN封了

三大运营商都这样....
被运营商强行上了个 PCDN-5M 服务，但主路由后台显示的上传仅仅 17.6 GB （下载 597.3 GB....）
总之运营商坏事做尽

**Reply by Joker2184** [reply]
不是
机房有台147K开mc50M占满.jpg

**Reply by LinQingYuu** [reply]
>机房有台147K开mc50M占满.jpg

Report abuse....
